WORKSPACE.yaml

```diff
@@ -7,7 +7,8 @@ defaultArgs:
   jbMarketplacePublishTrigger: "false"
   publishToJBMarketplace: true
   localAppVersion: unknown
-  codeCommit: 30d694d1f41945af7f8d757d3ddf176b283752cb
+  codeCommit: 85a18060d91c11ff6de8ba44e02c6c31a5cd907c
+  codeQuality: insider # TODO: revert this value to "stable" before merging
```

Comment:
Just need to set this to `stable` and we can merge it!

Reply:
Is there any order in which I should merge this PR vs https://github.com/gitpod-io/openvscode-server/pull/347?

Reply:
Nope, I'll merge the openvscode-server PR now

Reply:
value updated, waiting for approval and I can merge